agent/health_endpoint.go

```diff
@@ -219,8 +219,8 @@ func (s *HTTPHandlers) healthServiceNodes(resp http.ResponseWriter, req *http.Re
 		return nil, nil
 	}
 
-	// TODO: handle this for all endpoints in parseConsistency
-	args.QueryOptions.UseCache = s.agent.config.HTTPUseCache && args.QueryOptions.UseCache
+	useStreaming := s.agent.config.UseStreamingBackend && args.MinQueryIndex > 0
```

Comment:
It would be nice to capture a TODO for post GA to make this reloadable in config. I think that would give operators more confidence in experimenting with streaming in production if they could revert it without a rolling restart of all their agents.

Reply:
Sounds good! Added it to the project plan